src/components/FacetResults.js

```diff
@@ -57,6 +62,7 @@ type FacetResultsDefaultProps = {
   orderHint: Array<string>;
   entityColors: Map<string, string>;
   showEmptyFacets: boolean;
+  hideFacet: Array<string>,
```

Comment:
Rename prop to indicate this is an array not a boolean.
```suggestion
  hiddenFacets: Array<string>,
```